examples/client.go

```diff
@@ -0,0 +1,63 @@
+/*
+Wrapping the client can be useful when you want to override default behavior, such as always setting
```

Comment:
I wasn't sure if we wanted to retain this comment in the root readme, create a new readme in the examples dir?

Reply:
lets go with this for now - if someone is digging through example code having info inline like this is probably what they'd expect